### PR TITLE
schemadiff: Schema.Entity(), ToStatements(), ToQueries(), ToSQL()

### PR DESCRIPTION
**Pull request by shlomi-noach**
## Description

Usability functions for `Schema`: export to SQL, to statements ; get entity by name

## Checklist

-   [ ] "Backport me!" label has been added if this change should be backported
-   [x] Tests were added or are not required
-   [ ] Documentation was added or is not required

cc @dbussink

### CODE REVIEW

**Comment by shlomi-noach** [comment]
Whoops, missing semicolon in ToSQL.

File: go/vt/schemadiff/schema.go [hunk 1]
```diff
@@ -282,3 +282,31 @@ func (s *Schema) Diff(other *Schema, hints *DiffHints) (diffs []EntityDiff, err
...
+	queries := []string{}
+	for _, e := range s.Entities() {
+		queries = append(queries, e.Create().CanonicalStatementString())
+	}
+	return queries
+}
+
+// ToSQL returns a SQL blob with ordered sequence of queries which can be applied to create the schema
+func (s *Schema) ToSQL() string {
+	return strings.Join(s.ToQueries(), "\n")
```
Comment (dbussink) [inline go/vt/schemadiff/schema.go:311]: @shlomi-noach Was about to ask if this would need a `;\n` as a separator instead :smile:.

Comment (shlomi-noach) [reply go/vt/schemadiff/schema.go]: Now with semicolon

**Comment by shlomi-noach** [comment]
Implicitly merged by https://github.com/vitessio/vitess/pull/10183

**Comment by shlomi-noach** [comment]
Tracking issue: https://github.com/vitessio/vitess/issues/10203

**Pull request closed without merging**